### ISSUE TITLE
Change units of PSF table

### ISSUE THREAD

**Issue by jknodlseder** [open-gamma-ray-astro/gamma-astro-data-formats#54]
The units of the PSF table should be `sr^-1` which is the SI unit of a solid angle (see e.g. https://en.wikipedia.org/wiki/Steradian). Square degrees is a non-SI unit

**Comment by cdeil** [comment]
> The units of the PSF table should be sr^-1 which is the SI unit of a solid angle (see e.g. https://en.wikipedia.org/wiki/Steradian). Square degrees is a non-SI unit

We're using `deg` in many other places, so I don't see how using `deg^-2` could be bad.
@jknodlseder - Is `deg` also not an SI unit and you want to get rid of it?

But [gtpsf](http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/psf/psf_gtpsf/index.html) uses `sr^-1`, so for consistency, I'm 👍  to change the [psf_table](http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/psf/psf_table/index.html) spec to use `sr^-1`.
I can do the change in the spec.

@lmohrmann or @dltiziani Can you make the change in the HAP exporter?

**Comment by cdeil** [comment]
@dltiziani - Another thing to fix in the HESS exporter is this:
```
In [1]: from astropy.table import Table

In [2]: t = Table.read('datasets/hess-crab4-hd-hap-prod2/run023400-023599/run023523/hess_psf_table_023523.fits.gz')
WARNING: UnitsWarning: '1/(deg^2)' did not parse as fits unit: Numeric factor not supported by FITS [astropy.units.core]
```
We had this issue before ... the problem is the "1/" in the string ... the solution is to write "sr^-1", i.e. don't have a "1/" in the unit string.

**Comment by cdeil** [comment]
I've implemented this change in #55. Closing this issue now.
